QtlConnpassAPI/View/SearchView.swift

```diff
@@ -0,0 +1,130 @@
+//
+//  ContentView.swift
+//  ConnpassShowList
+//
+//  Created by 柿崎逸 on 2023/03/25.
+//
+
+import SwiftUI
+import CoreData
+
+struct SearchView: View {
+    // MARK: - Property Wrappers
+    @State var keyword = ""
+    // MARK: - Property Wrappers
+    @State private var isListView = false
+    // MARK: - Property Wrappers
+    @State private var isSearchAlert = false
+    // MARK: - Property Wrappers
+    @State private var isLogSearchAlert = false
+    // MARK: - Property Wrappers
+    @StateObject var searchViewModel = SearchViewModel()
+    // MARK: - Property Wrappers
+    @FetchRequest(sortDescriptors: []) var logs: FetchedResults<Log>
+    // MARK: - Property Wrappers
+    @Environment(\.managedObjectContext) var moc
```

Comment:
```suggestion
    // MARK: - Property Wrappers
    @State var keyword = ""
    @State private var isListView = false
    @State private var isSearchAlert = false
    @State private var isLogSearchAlert = false
    @StateObject var searchViewModel = SearchViewModel()
    @FetchRequest(sortDescriptors: []) var logs: FetchedResults<Log>
    @Environment(\.managedObjectContext) var moc
```

セクションのまとめで良いのでMARKコメントは一個あれば良いです！

Reply:
commit: 67acccf4b7c270fbdf610bca5e0bdb5f1ec86499
ここの修正を複数ファイルでcommitしていたので次から分けます(汗)